srcpkgs/python3-pypdf/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'python3-pypdf'
+pkgname=python3-pypdf
+version=3.4.0
+revision=1
+_sample_files_gitrev=372294b066cd3fbb4fb12fd9000ef935a2a86fcf
+build_wrksrc="pypdf-${version}"
+build_style=python3-pep517
+hostmakedepends="python3-flit_core"
+depends="python3-Pillow"
+checkdepends="python3-pytest python3-pycryptodome ${depends}"
+short_desc="PDF library for splitting, merging, cropping, and transforming pages"
+maintainer="Piotr Wójcik <chocimier@tlen.pl>"
+license="BSD-3-Clause"
+homepage="https://pypdf.rtfd.io/"
```

Comment:
where is this site linked from?

Reply:
The link under the `About` section of https://github.com/py-pdf/pypdf contains the readthedocs site but with hardcoded language version. It's favorable to use rtfd as it will always redirect to the default version of the project whatever it might change to in the future: https://docs.readthedocs.io/en/stable/automatic-redirects.html#rtfd-io

Reply:
It says it behaves same as <*.readthedocs.io/>, tet;s stay with what upsteam uses.
3.4.1 is out.